### PR TITLE
Update zh-CN translation to latest version

### PR DESCRIPTION
**Pull request by mitian233**
Update zh-CN translation to latest version

### CODE REVIEW

**Comment by auroursa** [comment]
I have pushed a pull request that includes my translation suggestion and addresses the remaining untranslated strings.

**Comment by mitian233** [comment]
Merged translations from @auroursa . LGTM :)

**Comment by auroursa** [comment]
Looks good to me +1

**Review by auroursa — COMMENTED** [review]
Looks good to me

File: src/locale/locales/zh-CN/messages.po [hunk 1]
```diff
@@ -132,100 +132,100 @@ msgstr "已取消隐藏账户"
...
 #: src/view/screens/ProfileList.tsx:802
 msgid "Add a user to this list"
-msgstr "将用户添加至列表"
+msgstr "将用户新增至列表"
 
 #: src/view/screens/Settings/index.tsx:383
 #: src/view/screens/Settings/index.tsx:392
 msgid "Add account"
-msgstr "添加账户"
+msgstr "新增账户"
```
Comment (mitian233) [inline src/locale/locales/zh-CN/messages.po:148]:
```suggestion
msgstr "添加账户"
```

File: src/locale/locales/zh-CN/messages.po [hunk 2]
```diff
@@ -132,100 +132,100 @@ msgstr "已取消隐藏账户"
...
 
 #: src/view/com/composer/Composer.tsx:451
 msgid "Add link card:"
-msgstr "添加链接卡片:"
+msgstr "新增链接卡片:"
 
 #: src/view/com/modals/ChangeHandle.tsx:417
 msgid "Add the following DNS record to your domain:"
-msgstr "将以下DNS记录添加到你的域名:"
+msgstr "将以下DNS记录新增到你的域名:"
```
Comment (mitian233) [inline src/locale/locales/zh-CN/messages.po:181]:
```suggestion
msgstr "将以下 DNS 记录新增到你的域名:"
```

File: src/locale/locales/zh-CN/messages.po [hunk 3]
```diff
@@ -3831,15 +3831,15 @@ msgstr "以下步骤将帮助定制你的 Bluesky 体验。"
...
+msgstr "此帖子可能已被删除。"
 
 #: src/view/screens/PrivacyPolicy.tsx:33
 msgid "The Privacy Policy has been moved to <0/>"
 msgstr "隐私政策已迁移至 <0/>"
 
 #: src/view/screens/Support.tsx:36
 msgid "The support form has been moved. If you need help, please <0/> or visit {HELP_DESK_URL} to get in touch with us."
-msgstr "支持表单已移动位置。如果你需要帮助，请点击<0/>或访问{HELP_DESK_URL}与我们联系。"
+msgstr "支持表单已迁移至别处。如果你需要帮助，请点击<0/>或访问{HELP_DESK_URL}与我们联系。"
```
Comment (mitian233) [inline src/locale/locales/zh-CN/messages.po:3842]:
```suggestion
msgstr "支持表单已被移除。如果你需要帮助，请点击<0/>或访问{HELP_DESK_URL}与我们联系。"
```

File: src/locale/locales/zh-CN/messages.po [hunk 4]
```diff
@@ -3310,7 +3310,7 @@ msgstr "选择以下一些账户进行关注"
 
 #: src/view/com/auth/server-input/index.tsx:82
 msgid "Select the service that hosts your data."
-msgstr ""
+msgstr "选择托管你数据的服务器。"
```
Comment (mitian233) [inline src/locale/locales/zh-CN/messages.po:3313]:
```suggestion
msgstr "选择服务器来托管你的数据。"
```

File: src/locale/locales/zh-CN/messages.po [hunk 5]
```diff
@@ -132,100 +132,100 @@ msgstr "已取消隐藏账户"
...
 
 #: src/view/com/modals/SelfLabel.tsx:56
 msgid "Add a content warning"
-msgstr "添加内容警告"
+msgstr "新增内容警告"
 
 #: src/view/screens/ProfileList.tsx:802
 msgid "Add a user to this list"
-msgstr "将用户添加至列表"
+msgstr "将用户新增至列表"
```
Comment (mitian233) [inline src/locale/locales/zh-CN/messages.po:143]:
```suggestion
msgstr "将用户添加至列表"
```

File: src/locale/locales/zh-CN/messages.po [hunk 6]
```diff
@@ -132,100 +132,100 @@ msgstr "已取消隐藏账户"
...
+msgstr "新增细节"
 
 #: src/view/com/modals/report/Modal.tsx:194
 msgid "Add details to report"
 msgstr "补充反馈详细内容"
 
 #: src/view/com/composer/Composer.tsx:446
 msgid "Add link card"
-msgstr "添加链接卡片"
+msgstr "新增链接卡片"
```
Comment (mitian233) [inline src/locale/locales/zh-CN/messages.po:173]:
```suggestion
msgstr "添加链接卡片"
```

File: src/locale/locales/zh-CN/messages.po [hunk 7]
```diff
@@ -132,100 +132,100 @@ msgstr "已取消隐藏账户"
...
 
 #: src/view/com/composer/Composer.tsx:451
 msgid "Add link card:"
-msgstr "添加链接卡片:"
+msgstr "新增链接卡片:"
```
Comment (mitian233) [inline src/locale/locales/zh-CN/messages.po:177]:
```suggestion
msgstr "添加链接卡片:"
```

File: src/locale/locales/zh-CN/messages.po [hunk 8]
```diff
@@ -132,100 +132,100 @@ msgstr "已取消隐藏账户"
...
 
 #: src/view/com/modals/ChangeHandle.tsx:417
 msgid "Add the following DNS record to your domain:"
-msgstr "将以下DNS记录添加到你的域名:"
+msgstr "将以下DNS记录新增到你的域名:"
 
 #: src/view/com/profile/ProfileHeader.tsx:309
 msgid "Add to Lists"
-msgstr "添加至列表"
+msgstr "新增至列表"
```
Comment (mitian233) [inline src/locale/locales/zh-CN/messages.po:185]:
```suggestion
msgstr "添加至列表"
```

File: src/locale/locales/zh-CN/messages.po [hunk 9]
```diff
@@ -132,100 +132,100 @@ msgstr "已取消隐藏账户"
...
 
 #: src/view/com/feeds/FeedSourceCard.tsx:243
 #: src/view/screens/ProfileFeed.tsx:272
 msgid "Add to my feeds"
-msgstr "添加至自定义信息流"
+msgstr "新增至自定义信息流"
```
Comment (mitian233) [inline src/locale/locales/zh-CN/messages.po:190]:
```suggestion
msgstr "添加至自定义信息流"
```

File: src/locale/locales/zh-CN/messages.po [hunk 10]
```diff
@@ -132,100 +132,100 @@ msgstr "已取消隐藏账户"
...
 
 #: src/view/com/auth/onboarding/RecommendedFeedsItem.tsx:139
 msgid "Added"
-msgstr "已添加"
+msgstr "已新增"
```
Comment (mitian233) [inline src/locale/locales/zh-CN/messages.po:194]:
```suggestion
msgstr "已添加"
```

File: src/locale/locales/zh-CN/messages.po [hunk 11]
```diff
@@ -132,100 +132,100 @@ msgstr "已取消隐藏账户"
...
 #: src/view/com/modals/ListAddRemoveUsers.tsx:191
 #: src/view/com/modals/UserAddRemoveLists.tsx:144
 msgid "Added to list"
-msgstr "添加至列表"
+msgstr "新增至列表"
 
 #: src/view/com/feeds/FeedSourceCard.tsx:125
 msgid "Added to my feeds"
-msgstr "添加至自定义信息流"
+msgstr "新增至自定义信息流"
```
Comment (mitian233) [inline src/locale/locales/zh-CN/messages.po:203]:
```suggestion
msgstr "添加至自定义信息流"
```

File: src/locale/locales/zh-CN/messages.po [hunk 12]
```diff
@@ -132,100 +132,100 @@ msgstr "已取消隐藏账户"
...
 #: src/view/com/auth/onboarding/RecommendedFeedsItem.tsx:139
 msgid "Added"
-msgstr "已添加"
+msgstr "已新增"
 
 #: src/view/com/modals/ListAddRemoveUsers.tsx:191
 #: src/view/com/modals/UserAddRemoveLists.tsx:144
 msgid "Added to list"
-msgstr "添加至列表"
+msgstr "新增至列表"
```
Comment (mitian233) [inline src/locale/locales/zh-CN/messages.po:199]:
```suggestion
msgstr "添加至列表"
```

File: src/locale/locales/zh-CN/messages.po [hunk 13]
```diff
@@ -132,100 +132,100 @@ msgstr "已取消隐藏账户"
...
+msgstr "调整回复中需要具有的喜欢数才会在你的信息流中显示。"
 
 #: src/view/com/modals/SelfLabel.tsx:75
 msgid "Adult Content"
 msgstr "成人内容"
 
 #: src/view/com/modals/ContentFilteringSettings.tsx:141
 msgid "Adult content can only be enabled via the Web at <0/>."
-msgstr "要显示成人内容，你必须访问网页端上的<0/>。"
+msgstr "要显示成人内容，你必须访问网页端<0/>启用。"
```
Comment (mitian233) [inline src/locale/locales/zh-CN/messages.po:215]:
```suggestion
msgstr "要显示成人内容，你必须访问网页端<0/>来启用。"
```

File: src/locale/locales/zh-CN/messages.po [hunk 14]
```diff
@@ -3113,24 +3113,24 @@ msgstr "请求变更"
 #: src/view/com/modals/ChangePassword.tsx:239
 #: src/view/com/modals/ChangePassword.tsx:241
 msgid "Request Code"
-msgstr "确认码"
+msgstr "请求码"
```
Comment (mitian233) [inline src/locale/locales/zh-CN/messages.po:3116]:
```suggestion
msgstr "确认码"
```

File: src/locale/locales/zh-CN/messages.po [hunk 15]
```diff
@@ -2156,7 +2156,7 @@ msgstr "登录未列出的账户"
 
 #: src/view/com/modals/LinkWarning.tsx:65
 msgid "Make sure this is where you intend to go!"
-msgstr "请确认！"
+msgstr "请确认这是你想去的地方！"
```
Comment (mitian233) [inline src/locale/locales/zh-CN/messages.po:2159]:
```suggestion
msgstr "请确认目标页面地址是否正确！"
```

File: src/locale/locales/zh-CN/messages.po [hunk 16]
```diff
@@ -2097,11 +2097,11 @@ msgstr "列表名称"
 
 #: src/view/screens/ProfileList.tsx:342
 msgid "List unblocked"
-msgstr "取消屏蔽列表"
+msgstr "解除屏蔽列表"
 
 #: src/view/screens/ProfileList.tsx:301
 msgid "List unmuted"
-msgstr "取消隐藏列表"
+msgstr "解除隐藏列表"
```
Comment (mitian233) [inline src/locale/locales/zh-CN/messages.po:2104]:
```suggestion
msgstr "解除对列表的隐藏"
```

File: src/locale/locales/zh-CN/messages.po [hunk 17]
```diff
@@ -2097,11 +2097,11 @@ msgstr "列表名称"
 
 #: src/view/screens/ProfileList.tsx:342
 msgid "List unblocked"
-msgstr "取消屏蔽列表"
+msgstr "解除屏蔽列表"
```
Comment (mitian233) [inline src/locale/locales/zh-CN/messages.po:2100]:
```suggestion
msgstr "解除对列表的屏蔽"
```

File: src/locale/locales/zh-CN/messages.po [hunk 18]
```diff
@@ -1993,7 +1993,7 @@ msgstr "了解有关 Bluesky 公开内容的更多详情。"
 
 #: src/view/com/modals/lang-settings/ContentLanguagesSettings.tsx:82
 msgid "Leave them all unchecked to see any language."
-msgstr "全部不选中以查看任何语言。"
+msgstr "全部留空以查看任何语言。"
```
Comment (mitian233) [inline src/locale/locales/zh-CN/messages.po:1996]:
```suggestion
msgstr "全部留空以查看所有语言的帖子。"
```

File: src/locale/locales/zh-CN/messages.po [hunk 19]
```diff
@@ -1870,8 +1870,8 @@ msgid "Input the username or email address you used at signup"
 msgstr "输入注册时使用的用户名或电子邮箱"
 
 #: src/view/com/auth/create/Step2.tsx:271
-#~ msgid "Input the verification code we have texted to you"
-#~ msgstr "输入收到的短信验证码"
+msgid "Input the verification code we have texted to you"
+msgstr "输入我们发送到你手机的短信验证码"
```
Comment (mitian233) [inline src/locale/locales/zh-CN/messages.po:1874]:
```suggestion
msgstr "输入你收到的短信验证码"
```

File: src/locale/locales/zh-CN/messages.po [hunk 20]
```diff
@@ -1790,7 +1790,7 @@ msgstr "托管服务提供商"
 
 #: src/view/com/modals/InAppBrowserConsent.tsx:44
 msgid "How should we open this link?"
-msgstr "我们该如何打开此链接？"
+msgstr "我们该如何开启此链接？"
```
Comment (mitian233) [inline src/locale/locales/zh-CN/messages.po:1793]:
```suggestion
msgstr "我们该如何打开此链接？"
```

File: src/locale/locales/zh-CN/messages.po [hunk 21]
```diff
@@ -1317,27 +1317,27 @@ msgstr "启用此设置以仅查看你关注的人之间的回复。"
...
+msgstr "为此应用专用密码命名"
 
 #: src/view/com/modals/VerifyEmail.tsx:105
 msgid "Enter Confirmation Code"
 msgstr "输入验证码"
 
 #: src/view/com/modals/ChangePassword.tsx:151
 msgid "Enter the code you received to change your password."
-msgstr "输入你收到的确认码以更改密码。"
+msgstr "输入你收到的验证码以更改密码。"
```
Comment (mitian233) [inline src/locale/locales/zh-CN/messages.po:1332]:
```suggestion
msgstr "输入你收到的确认码以更改密码。"
```

File: src/locale/locales/zh-CN/messages.po [hunk 22]
```diff
@@ -1162,44 +1162,44 @@ msgstr "双击以登录"
...
+msgstr "下载 CAR 文件"
 
 #: src/view/com/composer/text-input/TextInput.web.tsx:247
 msgid "Drop to add images"
-msgstr "拖放即可添加图片"
+msgstr "拖放即可新增图片"
 
 #: src/screens/Onboarding/StepModeration/AdultContentEnabledPref.tsx:111
 msgid "Due to Apple policies, adult content can only be enabled on the web after completing sign up."
 msgstr "受 Apple 政策限制，成人内容只能在完成注册后在网页端启用显示。"
```
Comment (mitian233) [inline src/locale/locales/zh-CN/messages.po:1178]:
```suggestion
msgstr "受 Apple 政策限制，显示成人内容只能在完成注册后在网页端设置中启用。"
```

File: src/locale/locales/zh-CN/messages.po [hunk 23]
```diff
@@ -721,7 +721,7 @@ msgstr "关闭导航页脚"
 
 #: src/view/shell/index.web.tsx:50
 msgid "Closes bottom navigation bar"
-msgstr "关闭底栏"
+msgstr "关闭底部导航列"
```
Comment (mitian233) [inline src/locale/locales/zh-CN/messages.po:724]:
```suggestion
msgstr "关闭底部导航栏"
```

**Review by mitian233 — COMMENTED** [review]
Discussion of the word "add" 's translations: "新增" and "添加" should be distinguished

File: src/locale/locales/zh-CN/messages.po [hunk 24]
```diff
@@ -1162,44 +1162,44 @@ msgstr "双击以登录"
...
 #: src/view/screens/Settings/ExportCarDialog.tsx:59
 #: src/view/screens/Settings/ExportCarDialog.tsx:63
 msgid "Download CAR file"
-msgstr ""
+msgstr "下载 CAR 文件"
 
 #: src/view/com/composer/text-input/TextInput.web.tsx:247
 msgid "Drop to add images"
-msgstr "拖放即可添加图片"
+msgstr "拖放即可新增图片"
```
Comment (mitian233) [inline src/locale/locales/zh-CN/messages.po:1174]:
```suggestion
msgstr "拖放即可添加图片"
```

**Review by mitian233 — COMMENTED** [review]
LGTM

**Comment by auroursa** [comment]
LGTM.

**Review by auroursa — COMMENTED** [review]
LGTM.

File: src/locale/locales/zh-CN/messages.po [hunk 25]
```diff
@@ -758,7 +758,7 @@ msgstr "完成引导并开始使用你的账户"
 
 #: src/view/com/auth/create/Step3.tsx:73
 msgid "Complete the challenge"
-msgstr ""
+msgstr "完成人机验证"
```
Comment (mitian233) [inline src/locale/locales/zh-CN/messages.po:761]:
```suggestion
msgstr "完成验证"
```

File: src/locale/locales/zh-CN/messages.po [hunk 26]
```diff
@@ -1370,7 +1370,7 @@ msgstr "输入你的用户名和密码"
 
 #: src/view/com/auth/create/Step3.tsx:67
 msgid "Error receiving captcha response."
-msgstr ""
+msgstr "人机验证响应错误"
```
Comment (mitian233) [inline src/locale/locales/zh-CN/messages.po:1373]:
```suggestion
msgstr "Captcha 响应错误"
```

File: src/locale/locales/zh-CN/messages.po [hunk 27]
```diff
@@ -2755,23 +2755,23 @@ msgstr "请设置你的密码。"
 
 #: src/view/com/auth/create/state.ts:131
 msgid "Please complete the verification captcha."
-msgstr ""
+msgstr "请通过人机验证"
```
Comment (mitian233) [inline src/locale/locales/zh-CN/messages.po:2758]:
```suggestion
msgstr "请完成 Captcha 验证"
```

**Comment by mitian233** [comment]
This PR is ready to be merged:)

File: src/locale/locales/zh-CN/messages.po [hunk 28]
```diff
@@ -1870,8 +1870,8 @@ msgid "Input the username or email address you used at signup"
 msgstr "输入注册时使用的用户名或电子邮箱"
 
 #: src/view/com/auth/create/Step2.tsx:271
-#~ msgid "Input the verification code we have texted to you"
-#~ msgstr "输入收到的短信验证码"
+msgid "Input the verification code we have texted to you"
```
Comment (auroursa) [inline src/locale/locales/zh-CN/messages.po:1873]:
```suggestion
#~ msgid "Input the verification code we have texted to you"
```

File: src/locale/locales/zh-CN/messages.po (hunk 19 above)
Comment (auroursa) [inline src/locale/locales/zh-CN/messages.po:1874]:
```suggestion
#~ msgstr "输入我们发送到你手机的短信验证码"
```

**Review by auroursa — COMMENTED** [review]
#~ should not be removed.

File: src/locale/locales/zh-CN/messages.po [hunk 29]
```diff
@@ -204,28 +204,28 @@ msgstr "添加至自定义信息流"
```
Comment (rikumi) [inline src/locale/locales/zh-CN/messages.po:204]: 应为“已添加至自定义信息流”。

File: src/locale/locales/zh-CN/messages.po [hunk 30]
```diff
@@ -2028,44 +2028,44 @@ msgstr "亮色"
...
 #: src/view/screens/PostLikedBy.tsx:27
 #: src/view/screens/ProfileFeedLikedBy.tsx:27
 msgid "Liked By"
-msgstr "点赞"
+msgstr "喜欢"
 
 #: src/view/com/feeds/FeedSourceCard.tsx:277
 msgid "Liked by {0} {1}"
-msgstr "{0} {1} 点赞"
+msgstr "{0} 个{1} 喜欢"
```
Comment (auroursa) [inline src/locale/locales/zh-CN/messages.po:2048]:
```suggestion
msgstr "{0} 个 {1} 喜欢"
```

File: src/locale/locales/zh-CN/messages.po [hunk 31]
```diff
@@ -2028,44 +2028,44 @@ msgstr "亮色"
...
 
 #: src/view/screens/ProfileFeed.tsx:606
 msgid "Liked by {likeCount} {0}"
-msgstr "{likeCount} {0} 点赞"
+msgstr "{likeCount} 个{0} 喜欢"
```
Comment (auroursa) [inline src/locale/locales/zh-CN/messages.po:2052]:
```suggestion
msgstr "{likeCount} 个 {0} 喜欢"
```

**Review by auroursa — COMMENTED** [review]
add spaces

**Review by auroursa — COMMENTED** [review]
looks good so far.

File: src/locale/locales/zh-CN/messages.po [hunk 32]
```diff
@@ -3553,7 +3553,7 @@ msgstr "在关注信息流中显示回复"
 
 #: src/view/screens/PreferencesHomeFeed.tsx:70
 msgid "Show replies with at least {value} {0}"
-msgstr "显示至少包含 {value} {0} 的回复"
+msgstr "显示至少包含 {value} 个{0} 的回复"
```
Comment (KiritaniAyaka) [inline src/locale/locales/zh-CN/messages.po:3556]:
```suggestion
msgstr "显示至少包含 {value} 个 {0} 的回复"
```

Maybe spacing is needed here

File: src/locale/locales/zh-CN/messages.po [hunk 33]
```diff
@@ -2697,11 +2697,11 @@ msgstr "密码已更新！"
 
 #: src/Navigation.tsx:160
 msgid "People followed by @{0}"
-msgstr "被这些人所关注 @{0}"
+msgstr "被 @{0} 所关注的人"
```
Comment (KiritaniAyaka) [inline src/locale/locales/zh-CN/messages.po:2700]:
```suggestion
msgstr "@{0} 关注的人"
```

I think this expression is simpler and clear

**Review by auroursa — COMMENTED** [review]
okey, leaning towards not making changes for now. Let's wait for the next PR and gather enough discussion before making a decision. 
This PR process has been a bit lengthy. I'm inclined to merge this PR at first.

**Review by KiritaniAyaka — COMMENTED** [review]
Maybe "屏蔽词" is more literally than "隐藏词"

**Review by pfrazee — APPROVED** [review]
Thanks everyone! Merging

**Pull request merged**